Ginger/GingerCoreNET/UserCofig/UserProfile.cs

```diff
@@ -432,14 +432,18 @@ public void SaveUserProfile()
         {
             try
             {
+                Console.WriteLine("Before Agent mapping");
                 SaveRecentAppAgentsMapping();
+                Console.WriteLine("After Agent mapping");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)